docs/sql/statements/revoke.rst

```diff
@@ -19,18 +19,26 @@ Synopsis
 
   REVOKE { { DQL | DML | DDL | AL [,...] } | ALL [ PRIVILEGES ] }
   [ON {SCHEMA | TABLE | VIEW} identifier [, ...]]
-  FROM user_name [, ...];
+  FROM name [, ...];
+
+  REVOKE role_name_to_revoke [, ...] FROM name [, ...]
```

Comment:
The rendering looks a bit odd here with the grey?

<img width="863" alt="Screenshot 2023-12-29 at 15 03 59" src="https://github.com/crate/crate/assets/218003/508429bf-a698-436a-9949-7e3a22a210a1">